### PR TITLE
Check in container build code

### PR DESCRIPTION
**Pull request by hcho3**
Now it is possible to automatically build inference containers using the Jenkins CI pipeline.

The containers use the [MXNet Model Server](https://github.com/awslabs/mxnet-model-server) for model serving. (Despite the name, MMS is able to accommodate custom backend logic.)

Future items:
* Build GPU containers.
* Build BYOM (Bring Your Own Model) containers for MXNet and Tensorflow.
* Support batch prediction.

### CODE REVIEW

File: container/Dockerfile.cpu [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+RUN pip3 install --pre --no-cache-dir mxnet-model-server
+
+RUN pip3 install --no-cache-dir numpy scipy xlrd Pillow
+
+#RUN pip3 install --no-cache-dir dlr
+
+RUN mkdir -p /home/model-server && cd /home/model-server \
+    && git clone --recursive https://github.com/neo-ai/neo-ai-dlr \
+    && cd neo-ai-dlr && git checkout 7dd33f829062e19df3c0f175912199ad176da970 \
```
Comment (wweic) [inline container/Dockerfile.cpu]: why `7dd33f829062e19df3c0f175912199ad176da970`?

Comment (hcho3) [reply container/Dockerfile.cpu]: @wweic Let's keep it for now. I'll submit a follow-up PR where the Docker build will embed the latest DLR artifact into the container.

Comment (wweic) [reply container/Dockerfile.cpu]: OK. Please create an issue and follow up with a PR.

Comment (hcho3) [reply container/Dockerfile.cpu]: See #43

**Pull request merged**